tools/packaging/common/istio-iptables.sh

```diff
@@ -147,9 +148,10 @@ INBOUND_PORTS_INCLUDE=${ISTIO_INBOUND_PORTS-}
 INBOUND_PORTS_EXCLUDE=${ISTIO_LOCAL_EXCLUDE_PORTS-}
 OUTBOUND_IP_RANGES_INCLUDE=${ISTIO_SERVICE_CIDR-}
 OUTBOUND_IP_RANGES_EXCLUDE=${ISTIO_SERVICE_EXCLUDE_CIDR-}
+OUTBOUND_PORTS_EXCLUDE=${ISTIO_LOCAL_OUTBOUND_PORTS_EXCLUDE-}
```

Comment:
As a general comment, because of how kustomize works as well as mesh expansion we need to make sure the options
can be passed via environment variables.  On mesh expansion the CLI is hardcoded in the systemd unit - users are expected to edit 'sidecar.env'.  Also kustomization of env vars is far far easier than manipulating parts of a CLI.

So for 1.3 I am hoping we can remove the CLI options from this script, and just use env vars consistently. 
And maybe we can stop adding more options - and use the env only for new stuff.